From 9424467a5d1fb1f971076187c07c78a81e9cf661 Mon Sep 17 00:00:00 2001
From: Arjan van de Ven <arjan@linux.intel.com>
Date: Thu, 11 Feb 2016 11:07:54 -0600
Subject: [PATCH 20/22] no early modprobe

---
 kernel/kmod.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/kernel/kmod.c b/kernel/kmod.c
index 0277d121..20b5777f 100644
--- a/kernel/kmod.c
+++ b/kernel/kmod.c
@@ -557,7 +557,7 @@ int call_usermodehelper_exec(struct subprocess_info *sub_info, int wait)
 	DECLARE_COMPLETION_ONSTACK(done);
 	int retval = 0;
 
-	if (!sub_info->path) {
+	if (!sub_info->path || system_state == SYSTEM_BOOTING) {
 		call_usermodehelper_freeinfo(sub_info);
 		return -EINVAL;
 	}
-- 
2.11.0

